cmd/fyne/vendor.go

```diff
@@ -19,92 +9,72 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"regexp"
 	"strings"
+
+	"fyne.io/fyne"
 )
 
 const (
 	// goModFile is the go module file
 	goModFile = "go.mod"
 	// goModVendorFile is the vendor module file
 	goModVendorFile = "vendor/modules.txt"
-	// glwfMod is the glfw module
-	glwfMod = "github.com/go-gl/glfw/v3.3/glfw"
-	// glwfModSrcDir is the glfw dir containing the c source code to copy
-	glwfModSrcDir = "glfw"
+	// glfwModNew is the glfw module regexp for 3.3+
+	glfwModNew = "github.com/go-gl/glfw/(v(\\d+\\.)?(\\*|\\d)?(\\*|\\d+))/glfw"
```

Comment:
```suggestion
	glfwModNew = `github\.com/go-gl/glfw/(v(\d+\.)?(\*|\d)?(\*|\d+))/glfw`
```
alternatively if we won't be strict about version match:

`glfwModNew = "github.com/go-gl/glfw/(v[^/]+)/glfw"`

Reply:
Nice hint on the string formatting - that had escaped me :)